spec/features/admin_order_management_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe "Admin Order Managment" do
+  let(:admin) { create(:admin) }
+  let(:user) { create(:user) }
+  let!(:item) { create(:item, id: 1) }
+  let!(:order) { user.orders.create(cart: {"1"=> 2 } ) }
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside parentheses detected.